Sources/ProjectDescription/Product.swift

```diff
@@ -47,4 +47,7 @@ public enum Product: String, Codable, Equatable, Sendable {
     /// enable them by adding a command line tool target, a target dependency in
     /// the dependent targets, and the right build settings to use the macro executable.
     case macro
+    /// A Login Item
+    /// https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPSystemStartup/Chapters/CreatingLoginItems.html
+    case loginItem
```

Comment:
`ProjectDescription` products have generally had an actual product counterpart in Xcode. I feel somewhat opposed to breaking that mental model.

I feel we should rather consider this to be a dedicated property in `ProjectDescription.Target`, similar to how we added support for [privacy manifests](https://github.com/tuist/tuist/pull/6117). And that only if the login item can't already be modeled with existing Tuist primitives – what are the pieces of the login item that you were not able to compose with the current `ProjectDescription` API?

Tagging @tuist/community-core for wider feedback here.